src/App.js

```diff
@@ -44,21 +89,37 @@ const Home = () => {
         <HomeCategoriesComponent categories={Categories.results} />
       </div>
       <div className="Products">
-        <FeaturedProductsComponent products={FeaturedProducts.results} />
+        <FeaturedProductsComponent products={FeaturedProductsArray} categories={CategoriesArray} />
+        <button className="See-more-button" onClick={() => event()}>
+          View all products
+        </button>
       </div>
     </div>
   );
 };
 
 function App() {
+  const [showHome, setHomeShow] = useState(true);
+  const [showProducts, setProductsShow] = useState(false);
```

Comment:
we can avoid bugs by conditionally render based on just one state variable
```suggestion
  const [currentPage, setCurrentPage] = useState('Home');
```